pandas/core/window.py

```diff
@@ -850,7 +850,41 @@ def mean(self, *args, **kwargs):
         return self._apply('roll_mean', 'mean', **kwargs)
 
     _shared_docs['median'] = dedent("""
-    %(name)s median
+    Calculate the %(name)s median.
+
+    Parameters
+    ----------
+    **kwargs
+        Under Review.
+
+    Returns
+    -------
+    Series or DataFrame
+        Returned object type is determined by the caller of the %(name)s
+        object.
+
+    See Also
+    --------
+    Series.%(name)s : Calling object with Series data
+    DataFrame.%(name)s : Calling object with DataFrames
+    Series.median : Equivalent method for Series
+    DataFrame.median : Equivalent method for DataFrame
+
+    Examples
+    --------
+    The below example will show a rolling calculation with a window size of
+    three.
+
+    >>> import pandas as pd
+    >>> import numpy as np
+    >>> s = pd.Series(np.arange(5))
```

Comment:
Perhaps construct the series manually to avoid the reader having to think about arange:
```
s = pd.Series([0, 1, 2, 3, 4])
```

Reply:
That make sense. I've changed it from `arange` to a simple `list`